readthedocs/projects/tasks/builds.py

```diff
@@ -220,12 +220,21 @@ class UpdateDocsTask(SyncRepositoryMixin, Task):
     max_retries = 5  # 5 per normal builds, 25 per concurrency limited
     default_retry_delay = 7 * 60
 
-    # Expected exceptions that will be logged as info only and not retried
+    # Expected exceptions that will be logged as info only and not retried.
+    # These exceptions are not sent to Sentry either because we are using
+    # ``SENTRY_CELERY_IGNORE_EXPECTED=True``.
+    #
+    # All exceptions generated by a user miss-configuration should be listed
+    # here. Actually, every subclass of ``BuildUserError``.
```

Comment:
Is this future proof too? Like, any subclasses of BuildUserError are already included here with the superclass in the `throws` list?

Reply:
This is 100% clear to me. Sentry docs does not specify if indicating the superclass is enough to avoid sending their subclasses as an error as well. So, I'm specifying all of them for now.

Reply:
Also, not sure if there is a good way to test it. However, we want _all_ `BuildUserError` to not be reported to Sentry because we know it's not our fault. The "problem" is that we need to specify _all_ the classes here, which may be a little tedious, but that's fine, I guess.